app/src/lib/status-parser.ts

```diff
@@ -23,6 +23,18 @@ export interface IStatusEntry {
   readonly oldPath?: string
 }
 
+export function isIStatusHeader(
```

Comment:
:art: I'd just name this `isStatusHeader` as the `I` on the interface is just a marker